crates/threshold-bls-ffi/src/ffi.rs

```diff
@@ -696,7 +708,6 @@ pub struct Keys {
 }
 
 #[derive(Clone)]
-#[repr(C)]
```

Comment:
Is there a reason this `#[repr(C)]` attribute is removed?

Reply:
oops, we should not remove it. Reverted.